app/app.js

```diff
@@ -4,14 +4,58 @@ angular
   .module( 'app', [
     'ngConstants',
     'ngSanitize',
+    'ngAnimate',
     'templateCache',
     'lnCms',
     'lnFilters',
     'lnPatterns',
   ])
+  .animation( '.view-content', appAnimation )
   .config( appConfig )
   .run( appRun );
 
+//
+// ANIMATION
+//
+
+appAnimation.$inject = ['$rootScope', '$state'];
+
+function appAnimation( $rootScope, $state ) {
+  function resetTweenProps( element, done ) {
+    //this is needed to make fixed elements work after the animations
+    TweenMax.set( element, {
+      clearProps: 'all',
+    });
+
+    done();
+  }
+
+  return {
+    enter: function enter( element, done ) {
+      if ( $state.current.name === 'loading' ) {
+        TweenMax.from( element, 1, {
```

Comment:
Can we just create an object with the options instead such as: 

```js
var options = {
   opacity: 0,
   onComplete: function complete() {
      resetTweenProps( element, done );
   },
};
if ( $state.current.name !== 'loading' ) {
  options.y = '-100'; 
}

TweenMax.from( element, options );
```

That will allow to remove the `else` block.